tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -86,29 +85,11 @@ def format_message(self, alert):
         return "Subject: [Alert] %s, %s threshold breached\n\n%s" % (
             alert.resource, alert.severity, alert.tags['message'])
 
-    def get_alert_destinations(self, alert):
```

Comment:
i have @anivargi gave a path like email enable notification, for now he told me to get that from user dir, later it modified to get from indexes